tests/streamlit_ui_tests/features/column_control_panel.spec.ts

```diff
@@ -0,0 +1,127 @@
+        await checkColumnCellsHaveExpectedValues(mito, 0, ['1970-01-01 00:00:01', '1970-01-01 00:00:04', '1970-01-01 00:00:07', '1970-01-01 00:00:10']);
+    })
+
+    test('Adding filter', async ({ page }) => {
+        const mito = await getMitoFrameWithTestCSV(page);
+        
+        await openColumnControlPanel(mito, 'Column1');
+        
+        await mito.getByText('Add Filter').click();
+        await mito.getByText('Add a Filter').click();
+        await mito.locator('.spacing-row', { hasText: 'Where' }).locator('input').fill('4');
+        await awaitResponse(page);
+
+        await checkColumnCellsHaveExpectedValues(mito, 0, ['7', '10']);
+    });
+
+    test('Sorting', async ({ page }) => {
+        const mito = await getMitoFrameWithTestCSV(page);
+        
+        await openColumnControlPanel(mito, 'Column1');
+    
+        await mito.getByText('Descending').click();
+        await awaitResponse(page);
+
+        await checkColumnCellsHaveExpectedValues(mito, 0, ['10', '7', '4', '1']);
+
+        await mito.getByText('Ascending').click();
+        await awaitResponse(page);
+
+        await checkColumnCellsHaveExpectedValues(mito, 0, ['1', '4', '7', '10']);
+    });
+
+    test('Change the number type', async ({ page }) => {
+        const mito = await getMitoFrameWithTestCSV(page);
+        await openColumnControlPanel(mito, 'Column1');
+
+        await changeNumType(mito, page, 'Currency');
+        await checkColumnCellsHaveExpectedValues(mito, 0, ['$1', '$4', '$7', '$10']);
+
+        await changeNumType(mito, page, 'Accounting');
+        await checkColumnCellsHaveExpectedValues(mito, 0, ['$1', '$4', '$7', '$10']);
+
+        await changeNumType(mito, page, 'Percent');
+        await checkColumnCellsHaveExpectedValues(mito, 0, ['100%', '400%', '700%', '1,000%']);
+
+        await changeNumType(mito, page, 'Scientific Notation');
+        await checkColumnCellsHaveExpectedValues(mito, 0, ['1e+0', '4e+0', '7e+0', '1e+1']);
+    });
+
+    test('Values tab: filtering values by clicking on them', async ({ page }) => {
+        const mito = await getMitoFrameWithTestCSV(page);
+        await openColumnControlPanel(mito, 'Column1');
+
+        await mito.locator('.control-panel-taskpane-tab', { hasText: 'Values'}).click();
+        await checkValuesTabHasExpectedValues(mito, ['1', '4', '7', '10']);
+
+        await mito.locator('.multi-toggle-box-row', { hasText: '4' }).locator('input[type="checkbox"]').click();
+        await awaitResponse(page);
+        await checkColumnCellsHaveExpectedValues(mito, 0, ['1', '7', '10']);
+    })
+
+    test('Loads summary stats', async ({ page }) => {
+        const mito = await getMitoFrameWithTestCSV(page);
+        await openColumnControlPanel(mito, 'Column1');
+
+        await mito.locator('.control-panel-taskpane-tab', { hasText: 'Stats'}).click();
+        await expect(mito.locator('.column-describe-table-row', { hasText: 'mean' })).toContainText('5.5')
```

Comment:
The column summary graph is loading for me on JupyterLab safari, can you double check that its not working on streamlit and then open an issue if so